app/models/food.rb

```diff
@@ -4,4 +4,8 @@ class Food < ApplicationRecord
   has_many :inventories, through: :inventory_foods, dependent: :destroy
   has_many :recipe_foods, dependent: :destroy
   has_many :recipes, through: :recipe_foods, dependent: :destroy
+
+  validates :name, presence: true, length: { maximum: 250 }
+  validates :measurement_unit, presence: true, comparison: { greater_than_or_equal_to: 0 }
```

Comment:
Great validations. It would be better if you remove `comparison: { greater_than_or_equal_to: 0 }` from 
`validate :measurement_unit ` `line 9` because it is not a numeric value it is a string value.